src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -509,12 +510,12 @@ public function testDontRunAlternativeNamespaceName()
         $tester = new ApplicationTester($application);
         $tester->run(['command' => 'foos:bar1'], ['decorated' => false]);
         $this->assertSame('
-                                                          
-  There are no commands defined in the "foos" namespace.  
-                                                          
-  Did you mean this?                                      
-      foo                                                 
-                                                          
+
+  There are no commands defined in the "foos" namespace.
+
+  Did you mean this?
+      foo
+
```

Comment:
This change is probably going to break tests (yes, it's annoying)

Reply:
Yes, I was fixing it :)